### PR TITLE
Feature/jupyterlabv2.0

### PR DESCRIPTION
**Pull request by BobCashStory**

### CODE REVIEW

**Comment by yuvipanda** [comment]
This looks great! Can you bump the versions up to 2.0, and then I can merge?

Thank you!

**Comment by BobCashStory** [comment]
@yuvipanda done !

File: jupyterlab-launcher-shortcuts/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "jupyterlab-launcher-shortcuts",
-  "version": "1.0.0-beta1",
+  "version": "1.0.1-alpha1",
```
Comment (yuvipanda) [inline jupyterlab-launcher-shortcuts/package.json:3]:
```suggestion
  "version": "2.0.0",
```

Comment (yuvipanda) [reply jupyterlab-launcher-shortcuts/package.json:3]: This too?

**Comment by BobCashStory** [comment]
<3

**Pull request merged**